lib/web_ui/lib/src/engine/skwasm/skwasm_impl/paragraph.dart

```diff
@@ -746,6 +746,10 @@ class SkwasmParagraphStyle extends SkwasmObjectWrapper<RawParagraphStyle> implem
       skStringFree(localeHandle);
     }
     paragraphStyleSetTextStyle(handle, textStyleHandle);
+
+    if (ui.ParagraphBuilder.shouldDisableRoundingHack) {
+      paragraphStyleSetApplyRoundingHack(handle, false);
+    }
```

Comment:
Not sure I understand this. Why not:
```suggestion
    paragraphStyleSetApplyRoundingHack(
      handle,
      !ui.ParagraphBuilder.shouldDisableRoundingHack,
    );
```

Reply:
Because there is no reason to invoke this API when `shoudlDisableRoundingHack` is false, since the rounding hack is true by default.